recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   noarch: python
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
```

Comment:
Need to update other package requirements, matching `pyproject.toml` in upstream repository